CMakeLists.txt

```diff
@@ -403,7 +403,7 @@ if (NOT CMAKE_CROSSCOMPILING)
             OUTPUT ${output_file}
             COMMAND ./openrct2-cli sprite build "${CMAKE_BINARY_DIR}/${output_file}" "${json_file}"
             WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
-            DEPENDS ${dependent_files}
+            DEPENDS ${json_file}
```

Comment:
This change looks unintentional, right?

Reply:
It is not. It is in a commit of its own, because it fixes a bug where renaming or removing a file from g2/graphics would cause a build failure. I can create a separate PR for it if you want, though.

Reply:
I think the intention behind the dependency line is that changing one file (whether the JSON or one of the PNGs) triggers a rebuild of the target DAT file. If that instead causes a crash, that sounds like an issue.

Perhaps @duncanspumpkin can weigh in here.

Reply:
I changed that myself, together with Duncan... it’s just an oversight.

Reply:
The correct fix would be L401 add CONFIGURE_DEPENDS after dependent_files.